cmd/kube-proxy/proxy.go

```diff
@@ -41,7 +42,8 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	if err := command.Execute(); err != nil {
+	if err := command.Execute(); err != nil {		
+		fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
better to use `fmt.Fprintf(os.Stderr, "error: %v\n", err)`?

`kube-proxy` is not the only one cmd that missing such error msg. Please update them as well.